tools/nni_cmd/launcher.py

```diff
@@ -525,7 +525,14 @@ def create_experiment(args):
 
     nni_config.set_config('experimentConfig', experiment_config)
     nni_config.set_config('restServerPort', args.port)
-    launch_experiment(args, experiment_config, 'new', config_file_name)
+    try:
+        launch_experiment(args, experiment_config, 'new', config_file_name)
+    except Exception as exception:
+        nni_config = Config(config_file_name)
+        restServerPid = nni_config.get_config('restServerPid')
+        if restServerPid:
+            kill_command(restServerPid)
+        print_error(exception)
```

Comment:
suggest to also add `exit` here

Reply:
This function is a top level function, no more code will be executed after this function. If there is error here, it will exit the process automatically.

Reply:
if there is an exception, the exit code should be non-zero.
if you catch here, even there is an exception, exit code is still 0.

Reply:
ok, fixed

Reply:
how about raising the error here instead of print_error?

Reply:
The output format of raising error is not elegant, NNICTL use uniform `print_error()` method for error information.